debug.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gcr.io/distroless/static-debian11:debug
```

Comment:
nit: mind if we rename this file to `Dockefile.debug` to keep the files organized next to one another?

Reply:
flipped